server/server.go

```diff
@@ -42,8 +42,9 @@ func NewConfiguredServer() *Server {
 		InterruptChan:  make(chan os.Signal),
 		DefaultHeaders: make(map[string]string),
 	}
-	s.DefaultHeaders["X-Access-Control-Allow-Origin"] = "*"
-	s.DefaultHeaders["Server"] = "lbry.tv"
+	s.DefaultHeaders["Access-Control-Allow-Origin"] = "*"
+	s.DefaultHeaders["Content-Type"] = "application/json; charset=utf-8"
+	s.DefaultHeaders["Server"] = "api.lbry.tv"
```

Comment:
This is just cosmetic/vanity [HTTP header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Server) and I believe it won't affect anything even if we put `youtube.com` there.